frontend/genebank_pedigree.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file This file contains the Pedigree function. This function fetches
+ *       pedigree for a given `id` (parsed from the url).
+ */
+
+import React, { Component, createRef } from 'react'
+import { Link } from "react-router-dom";
+import { get } from './communication';
+import { Network } from 'vis-network';
+import "vis-network/styles/vis-network.css"
+
+/**
+ * Shows the information of a given individual and the pedigree graph built using the vis-network component
+ */
+export function GenebankPedigree({ id }: { id: string }) {
+  const [pedigree, setPedigree] = React.useState(undefined as any)
+
+  React.useEffect(() => {
+
+    get(`/api/genebank_pedigree/${id}`).then(
+      data => data && setPedigree(data),
+      error => console.error(error)
+    )
+  }, [id])
+
+  class GenebankPedigreeNetwork extends Component {
```

Comment:
This is written in old-style class-based React which is not used elsewhere in the project.

Reply:
Mmmm, ok. This is my first react app. I will check it.

Reply:
solved

Reply:
Perfect! :+1: